apps/server/src/shared/domain/service/nexboard.service.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { FeathersServiceProvider } from '@shared/infra/feathers/feathers-service.provider';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { Logger } from '@src/core/logger';
+import { EntityId } from '@shared/domain';
+import { ObjectId } from '@mikro-orm/mongodb';
+import { NexboardService } from './nexboard.service';
+
+describe('Nexboard service', () => {
+	let module: TestingModule;
+	let nexboardService: NexboardService;
+	let feathersServiceProvider: DeepMocked<FeathersServiceProvider>;
+
+	beforeEach(async () => {
+		module = await Test.createTestingModule({
+			providers: [
+				NexboardService,
+				{
+					provide: FeathersServiceProvider,
+					useValue: createMock<FeathersServiceProvider>(),
+				},
+				{
+					provide: Logger,
+					useValue: createMock<Logger>(),
+				},
+			],
+		}).compile();
+
+		nexboardService = module.get(NexboardService);
+		feathersServiceProvider = module.get(FeathersServiceProvider);
+	});
+
+	describe('create Nexboard', () => {
+		it('it should call feathers service', async () => {
+			feathersServiceProvider
+				.getService('/nexboard/boards')
+				.create.mockResolvedValue({ id: '123', publicLink: 'boardId' });
+
+			const userId: EntityId = new ObjectId().toHexString();
+
+			const response = await nexboardService.createNexboard(userId, 'title', 'description');
+			if (response) {
```

Comment:
would it work to to `expect(response).toBeDefined()` first so that the if statement is not necessary anymore?

Reply:
no, linter complains because the response can be also false